lib/resources/auditd.rb

```diff
@@ -0,0 +1,236 @@
+# encoding: utf-8
+# copyright: 2015, Vulcano Security GmbH
```

Comment:
We can remove this line - the copyright and license are covered at a project-level.